packages/aws/src/RPA/Cloud/AWS/__init__.py

```diff
@@ -1126,6 +1847,8 @@ def __init__(self, region: str = DEFAULT_REGION, robocloud_vault_name: str = Non
         ServiceTextract.__init__(self)
         ServiceComprehend.__init__(self)
         ServiceSQS.__init__(self)
+        ServiceRedshiftData.__init__(self)
```

Comment:
Peculiar way of initialising the super classes, where a `super().__init__()` will take care of all the upper inits. (if we keep the idiom in all super-classes `__init__` methods) -- might be a good time for some opportunistic refactoring here.

Reply:
Good point, I hadn't thought about how `super()` obviously hits all of the inherited classes.